plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts

```diff
@@ -6,22 +6,52 @@ import { __ } from '@wordpress/i18n';
 // @todo: Implement the right copy for the pattern categories: https://github.com/woocommerce/woocommerce/issues/48381
 export const PATTERN_CATEGORIES = {
 	intro: {
-		label: __( 'intro', 'woocommerce' ),
+		label: __( 'Intro', 'woocommerce' ),
 		description: __(
 			'Welcome shoppers to your store with one of our introductory patterns.',
 			'woocommerce'
 		),
 	},
+	'featured-selling': {
+		label: __( 'Featured selling', 'woocommerce' ),
+		description: __(
+			'Put the spotlight on one or more of your products or product categories.',
+			'woocommerce'
+		),
+	},
 	about: {
-		label: __( 'about', 'woocommerce' ),
-		description: __( 'about', 'woocommerce' ),
+		label: __( 'About', 'woocommerce' ),
+		description: __(
+			'Show your shoppers what’s special about your business.',
+			'woocommerce'
+		),
 	},
 	services: {
-		label: __( 'services', 'woocommerce' ),
-		description: __( 'services', 'woocommerce' ),
+		label: __( 'Services', 'woocommerce' ),
+		description: __(
+			'Share information on any services that your business can provide.',
+			'woocommerce'
+		),
+	},
+	reviews: {
+		label: __( 'Reviews', 'woocommerce' ),
+		description: __(
+			'Encourage sales by sharing positive feedback from happy shoppers.',
+			'woocommerce'
+		),
 	},
-	testimonials: {
-		label: __( 'testimonials', 'woocommerce' ),
-		description: __( 'testimonials', 'woocommerce' ),
+	'social-media': {
+		label: __( 'Social media', 'woocommerce' ),
+		description: __(
+			'Promote your social channels and give shoppers a way to see your latest products and news.',
+			'woocommerce'
+		),
 	},
+	// newsletter: {
```

Comment:
Is there a particular reason why we keep this in the code?

Reply:
I think this is a category that we'll be adding eventually, so I thought about leaving it there for when it's added.

Reply:
Ok, thanks! 🙇